File: codeforces/cf835div4/A/A.cpp
#include <bits/stdc++.h>
using namespace std;

int n,m;
int a[2];
void solve() {
    cin >> a[0] >> a[1] >> a[2];
    sort(a,a+3);
    cout << a[1] << "\n";
}
int main(){
    // int a,b,c;
    int T;
    cin >> T;
    while(T--) {
        solve();
    }
    return 0;
}

File: codeforces/cf835div4/A/A.go
package main


import (
	"bufio"
	"fmt"
	"os"	 
	"sort"
)

var r = bufio.NewReader(os.Stdin)
var w = bufio.NewWriter(os.Stdout)

func solve() {
	// var a,b,c int
	a := make([]int,3)
	fmt.Fscan(r,&a[0],&a[1],&a[2])
	sort.Ints(a)
	
	fmt.Fprintln(w,a[1])
}
func main(){
	defer w.Flush()
	var n int
	fmt.Fscan(r,&n)
	for i := 1; i <= n; i++ {
	    solve()
	}
	
}